src/main/java/campus/infrastructure/Storage.java

```diff
@@ -0,0 +1,75 @@
+package campus.infrastructure;
+
+import java.nio.file.Files;
+import java.io.BufferedWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import campus.tasks.Task;
+
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file
+ */
+public class Storage {
+    private final String filePath;
+    private List<String> listOfStrings;
+
+    /**
+     * Initialiser for the Storage Class
+     * @param filePath The filePath as a String of the textfile used for storage
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+
+        try {
+            this.listOfStrings = this.readFromDBCreateIfNotExists(this.filePath);
+        } catch (FileNotFoundException e) {
+            System.err.println("Error: " + e.getMessage());
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public List<String> getListOfStrings() {
+        return this.listOfStrings;
+    }
+
+    /**
+     * Function to read from the expected file called "data", if not, create the file and return null
+     * @param filePath Takes in the relative path file that it expects the datafile to be at
+     * @return Returns a list of strings, each string contains information about the DB, +
+     *      null if initialised for the first time.
+     */
+    public List<String> readFromDBCreateIfNotExists(String filePath) throws IOException {
```

Comment:
Naming seems a little bit long. Could consider a more concise naming convention
```suggestion
    public List<String> createTaskList(String filePath) throws IOException {
```

Reply:
thank you for the suggestion! I was thinking about how to better name this as well but I feel that `createTaskList` doesnt quite capture what the function does, maybe I can change it to another name but throw the behaviour in the documentation.